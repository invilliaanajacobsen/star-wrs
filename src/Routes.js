import React from 'react'
import BrowserRouter from 'react-router-dom/BrowserRouter'
import Route from 'react-router-dom/Route'
import Switch from 'react-router-dom/Switch'
import LazyImport from './Lazy'

const Home = LazyImport(() => import('./Components/template/Home'))

const Routes = () => (
  <BrowserRouter>
    <Switch>
      <Route exact path="/" component={Home} />
    </Switch>
  </BrowserRouter>
)

export default Routes