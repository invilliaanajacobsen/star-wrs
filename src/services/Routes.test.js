import React from "react";
import { shallow } from "enzyme";

import Routes from "./Routes";

const initStateIfEmptyMock = jest.fn();

describe("Routes component", () => {
	it("should match snapshot", () => {
		expect(
			shallow(<Routes initStateIfEmpty={initStateIfEmptyMock} />)
		).toMatchSnapshot();
	});

	it("should invoke initStateIfEmpty", () => {
		shallow(<Routes initStateIfEmpty={initStateIfEmptyMock} />);
		expect(initStateIfEmptyMock).toHaveBeenLastCalledWith();
	});
});
