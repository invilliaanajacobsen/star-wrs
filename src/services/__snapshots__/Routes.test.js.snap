// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes component should match snapshot 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
  </Switch>
</BrowserRouter>
`;
